scp/cc/azure/attestation/src/print_report.cc

```diff
@@ -22,9 +22,11 @@ using google::scp::azure::attestation::fetchFakeSnpAttestation;
 using google::scp::azure::attestation::fetchSnpAttestation;
 using google::scp::azure::attestation::hasSnp;
 
-int main() {
-  const auto report =
-      hasSnp() ? fetchSnpAttestation() : fetchFakeSnpAttestation();
+int main(int argc, char* argv[]) {
+  std::string report_data = argc > 1 ? argv[1] : "";
+  const auto report = 
+    hasSnp() ? fetchSnpAttestation(report_data) : fetchFakeSnpAttestation();
```

Comment:
fetchFakeSnpAttestation() should also pass in report_data with the fixed hash of our wellknown test key.

Reply:
Did we reach a conclusion on "fake report needs to be checkable by CCF" vs "just stuff any old thing into the REPORT_DATA of a template report and don't worry that the signing is invalid"?

I think we went with the first and so would need a well known test key/hash that matches the fake report and then have the wrapping key be fixed on non SNP setups.

Reply:
This particular path is just for printing a report so I don't think we ever care about providing report data to the fake one.  Maybe we should not allow this tool to run at all on non SNP hardware?